src/main/java/org/terasology/potions/component/PotionEffect.java

```diff
@@ -17,10 +17,15 @@
 
 import org.terasology.reflection.MappedContainer;
 
-// This class or container contains details on a particular effect of a potion.
+/** This class or mapped container contains details on a particular effect of a potion. */
```

Comment:
Afaik, the mapped container allows to serialize the effect list. I'd just go for "Common class for describing a potion effect" or something like that.